spec/services/session_store_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+
+RSpec.describe SessionStore do
+  let(:session_uuid) { SecureRandom.uuid }
+
+  subject(:store) { described_class.new(session_uuid) }
+
```

Comment:
I think we probably want something like this around/flushdb block to make sure this test has a clean slate when it runs:

https://github.com/18F/identity-idp/blob/73e0c259d1dd6f9c590085e9d32c5e3815f520f8/spec/jobs/risc_delivery_job_spec.rb#L4-L8